packages/Webkul/Shop/src/Database/Migrations/2023_07_12_181722_remove_home_page_and_footer_content_column.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Change the migration file name from "remove_home_page_and_footer_content_column" to "remove_home_page_and_footer_content_column_from_channel_translations_table"

Reply:
done